workloads/templates/vm-creation.md

```diff
@@ -2,68 +2,110 @@
 
 ## Overview
 
-The [OpenShift's template mechanism](https://docs.openshift.org/latest/dev_guide/templates.html) allows user to create a set of objects from a template.  KubeVirt takes benefit from this template mechanism to create VirtualMachines.
+The KubeVirt projects provides a set of [templates](https://docs.okd.io/latest/dev_guide/templates.html) to create VMs to handle common usage scenarios.
+These templates provide a combination of some key factors that could be further customized and processed to have a Virtual Machine object.
+
+The key factors which define a template are
+ - Workload
+  Most Virtual Machine should be *generic* to have maximum flexibility; the *highperformance* workload trades some of this flexibility to
+  provide better performances.
+- Guest Operating System (OS)
+  This allow to ensure that the emulated hardware is compatible with the guest OS. Furthermore, it allows to maximize the stability
+  of the VM, and allows performance optimizations.
+- Size (flavor) 
+  Defines the amount of resources (CPU, memory) to allocate to the VM.
 
-In order to create a virtual machine via OpenShift CLI, you need to provide a template defining the corresponding object and its metadata.
 
-!> Only `VirtualMachine` object is currently supported.
+## WebUI
+Kubevirt project has [the official UI](https://github.com/kubevirt/web-ui). This UI supports creation VM using templates and templates features - flavors and workload profiles.
+To create VM from template, choose WorkLoads in the left panel >> press to the "Create Virtual Machine" blue button >> choose "Create from Wizzard". Next, you have to see "Create Virtual Machine" window
```

Comment:
```suggestion
To create VM from template, choose _WorkLoads_ in the left panel, press the _Create Virtual Machine_ blue button, choose _Create from Wizard_. Next, you will see the "Create Virtual Machine" window and you should be able to select the guest operating system, workload profile and size.
```

The spelling and formatting was really bad, you should not change styles within one sentence.

Reply:
I've add it in the latest commit
>The spelling and formatting
Sorry